package.json

```diff
@@ -23,17 +26,22 @@
     "test:ci:e2e": "testcafe 'firefox:headless,chrome:headless' tests/e2e/**/*.e2e.{j,t}s -c 2 -r xunit:/tmp/test-results/e2e.xml"
   },
   "dependencies": {
+    "@hapi/joi": "^17.1.0",
     "@quasar/extras": "^1.4.2",
     "@sentry/browser": "^5.11.2",
     "@sentry/integrations": "^5.11.1",
+    "aws-sdk": "^2.613.0",
     "axios": "~0.19.0",
     "binary-search-bounds": "^2.0.4",
     "csvtojson": "^2.0.10",
     "geolib": "^3.2.1",
+    "http-errors": "^1.7.3",
     "intl-messageformat-parser": "^2.1.3",
     "lodash": "^4.17.15",
     "mapbox-gl": "~1.7.0",
     "mapbox-gl-vue": "^2.0.4",
+    "middy": "^0.33.2",
```

Comment:
Does Netlify skip `devDependencies` during `npm install`? Same for `joi`, `http-errors`, `aws-sdk` and `mobitel-iso-4217-currencies`.

Reply:
I didn't find documentation about this.

However to make it clearer, I isolated Netlify functions' dependencies from others by creating another `package.json` into the folder `netlify` and added a `postinstall` script.